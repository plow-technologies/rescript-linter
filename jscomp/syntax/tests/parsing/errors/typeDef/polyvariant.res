type t = [< ]

type z = [< | #A | #B > ]

type rec t = []

