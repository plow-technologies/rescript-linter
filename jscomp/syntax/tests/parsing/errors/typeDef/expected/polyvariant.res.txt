
  Syntax error!
  tests/parsing/errors/typeDef/polyvariant.res:1:13

  1 │ type t = [< ]
  2 │ 
  3 │ type z = [< | #A | #B > ]

  I'm not sure what to parse here when looking at "]".


  Syntax error!
  tests/parsing/errors/typeDef/polyvariant.res:5:15

  3 │ type z = [< | #A | #B > ]
  4 │ 
  5 │ type rec t = []
  6 │ 
  7 │ 

  I'm not sure what to parse here when looking at "]".

type nonrec t = [< [%rescript.typehole ]]
type nonrec z = [< `A  | `B ]
type t = [ [%rescript.typehole ] | [%rescript.typehole ]]